Doc/library/select.rst

```diff
@@ -280,43 +280,47 @@ Edge and Level Trigger Polling (epoll) Objects
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLPRI`         | Urgent data for read                          |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLERR`         | Error condition happened on the assoc. fd     |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLHUP`         | Hang up happened on the assoc. fd             |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLET`          | Set Edge Trigger behavior, the default is     |
+   |                           | Level Trigger behavior                        |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLONESHOT`     | Set one-shot behavior. After one event is     |
+   |                           | pulled out, the fd is internally disabled     |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLEXCLUSIVE`   | Wake only one epoll object when the           |
+   |                           | associated fd has an event. The default (if   |
+   |                           | this flag is not set) is to wake all epoll    |
+   |                           | objects polling on a fd.                      |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLRDHUP`       | Stream socket peer closed connection or shut  |
+   |                           | down writing half of connection.              |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLRDNORM`      | Equivalent to :const:`EPOLLIN`                |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLRDBAND`      | Priority data band can be read.               |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLWRNORM`      | Equivalent to :const:`EPOLLOUT`               |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLWRBAND`      | Priority data may be written.                 |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLMSG`         | Ignored.                                      |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLL_URING_WAKE` | Optimizes event wake-up using io_uring.       |                                              |
+   +---------------------------+-----------------------------------------------+
+   | :const:`EPOLLWAKEUP`      | Prevents sleep during event waiting.          |                                              |
+   +---------------------------+-----------------------------------------------+
```

Comment:
```suggestion
   +---------------------------+-----------------------------------------------+
   | :const:`EPOLL_URING_WAKE` | Optimizes event wake-up using io_uring.       |
   +---------------------------+-----------------------------------------------+
   | :const:`EPOLLWAKEUP`      | Prevents sleep during event waiting.          |
   +---------------------------+-----------------------------------------------+
```

You seem to have additional `|` at the end.

Reply:
Sorry. I'm addicted to space key ;)